assets/i18n/qqq.json

```diff
@@ -484,6 +484,7 @@
 	"I18N_INTERACTIONS_SET_INPUT_NO_ANSWER": "Text displayed in a set interaction. - The interaction is a dialog between Oppia and the user. This text is shown on the user side of the dialog if the set introduced by the user has no elements.",
 	"I18N_INTERACTIONS_SUBMIT": "Text displayed in an interaction or navigation bar. - The text is shown inside a button. When the user clicks the button, the system checks if the answer is correct and continues to the following action.\n{{Identical|Submit}}",
 	"I18N_INTERACTIONS_TERMS_LIMIT": "Error message shown below the input field for the ratio input interaction.",
+	"I18N_INVALID_TAGS_AND_ATTRIBUTES_ALERT": "Alert message shown when invalid tags and attributes have been stripped off from the uploaded SVG",
```

Comment:
stripped off --> stripped

Reply:
Hi! Sorry, I am not clear of what the truncate part refers to? Should we shorten the alert message shown to the user?

Reply:
Done